cypress/README.md

```diff
@@ -43,7 +43,7 @@ The default Cypress configuration can be customized by creating a `cypress.env.j
 
 ```json
 {
-  "baseUrl": "http://localhost:30001"
+  "baseUrl": "http://localhost:8080"
```

Comment:
I may have missed it but I assume NATing is being done somewhere to bring this to 30001?

Reply:
```
--publish 30000:30001/tcp
```